Cartfile

```diff
@@ -1,8 +1,8 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.6
-github "mapbox/MapboxDirections.swift" ~> 0.10
+github "mapbox/MapboxDirections.swift" "master"
```

Comment:
Can we wait for a release? Compiling via cocoapods will fail until we update the podspec.

Reply:
Of course, that’s why I mentioned that this PR depends on mapbox/MapboxDirections.swift#174. I only put this line in here to verify that tests pass.

Reply:
This PR now depends on v0.10.5.